doc/source/rllib/rllib-rlmodule.rst

```diff
@@ -31,33 +36,60 @@ RL Modules (Alpha)
          - |pytorch| |tensorflow|
          -
-         - |pytorch|
        * - **APPO**
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
          - |pytorch| |tensorflow|
-         - 
-         - 
+         -
+       * - **SAC**
+         -
+         - |pytorch|
+         -
+         -
+
 
+RL Module is a neural network container that implements three public methods:
+:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_train`,
+:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration`,
+and :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference`.
+Each method corresponds to a distinct reinforcement learning and/or production phase.
 
+The :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference` method computes
+actions in a purely greedy fashion. It is normally used only in production settings (after
+training has been completed) or for evaluation purposes parallel to training.
+RLlib's :py:class:`~ray.rllib.evaluation.env_runner.EnvRunner` classes will chose this method
+if the `explore` flag in the :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig`
+is set to False.
 
-RL Module is a neural network container that implements three public methods: :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_train`, :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration`, and :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference`. Each method corresponds to a distinct reinforcement learning phase.
+:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration` computes actions
+in an exploratory way, balancing exploration and exploitation. It is the method normally used
+for training data collection by RLlib's :py:class:`~ray.rllib.evaluation.env_runner.EnvRunner` classes
+(if the `explore` flag in the :py:class:`~ray.rllib.algorithms.algorithm_config.AlgorithmConfig`
+is set to True, which it is by default).
+However, note that this method can also be used after training in an inference-only phase, if the user
+explicitly want the computed actions to be exploratory. Some environments - like the rock-paper-scissors game -
+can only be optimally solved by acting stochastically.
 
-:py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_exploration` handles acting and data collection, balancing exploration and exploitation. On the other hand, the :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_inference` serves the learned model during evaluation, often being less stochastic.
+Finally, the :py:meth:`~ray.rllib.core.rl_module.rl_module.RLModule.forward_train` method is called
+during the training update phase of an Algorithm and not only should it compute actions (or action logits/probabilities),
+but anything that is relevant and needed for the algorithm's loss function, such as computing
+Q-values in a DQN model or value function predictions in a PPO model.
```

Comment:
would rather name it "target Q-values in a DQN ..." as for exploration and inference we nee Q-values, too.